cli-commands/compiler/compiler.js

```diff
@@ -54,6 +56,15 @@ const performCompilation = async (defaultPath, runs, solcVersion, useDocker, qui
 		}
 	}
 
+	if (deleteCompiledFiles) {
```

Comment:
why use nested ifs, instead of just `&&`?

Also isnt there a way to reverse the logic in order to completely avoid the nesting?